distributions/test/src/test/java/com/connexta/replication/distributions/test/TestUtils.java

```diff
@@ -86,7 +86,7 @@ public static GenericContainer defaultSolrContainer(Network network) {
         .withNetwork(network)
         .withNetworkAliases("replication-solr")
         .withExposedPorts(SOLR_PORT)
-        .waitingFor(Wait.forHttp("/solr/replication_" + FILTER + "/admin/ping"));
+        .waitingFor(Wait.forHttp("/solr/replication_" + SITE + "/admin/ping"));
```

Comment:
Wanted to wait for the site core instead since it's the first core we call to.

Reply:
I want to but test containers doesn't provide a way to do that. I could look into a solution but that will be quite a bit of work and I don't want to put anymore effort into it unless it becomes a problem.